resources/views/guest/mainPage.blade.php

```diff
@@ -0,0 +1,255 @@
+                <form class="form-inline my-2 my-lg-0">
+                    <a href="" class="btn  mr-sm-2 text-white nav-font mx-1"><i class="fas fa-sign-in-alt"></i> Log in</a>
+                    <a href="" class="btn btn-success my-2 my-sm-0" type="submit"><i class="fa fa-user mx-1"></i>Employer?</a>
+                </form>
+            </div>
+        </nav>
+
+              {{-- Search A Job --}}
+        <div class="container search-container">
+            <div class="form-group search row ">
+                <div class="col-lg-12 mb-3 ">
+                    <h2 class="mx-auto text-white font-weight-bolder d-flex justify-content-center">Find the Best Jobs in Egypt</h2>
+                    <h5 class="mx-auto text-white  font-weight-bolder my-3 d-flex justify-content-center">Searching for vacancies & career opportunities? WUZZUF helps you in your job search in Egypt
+                    </h5>
+                </div>
+                <div class="col-lg-12">
+                   <form action="">
+                       <div class="d-flex">
+                        <div class="text-center job-search-container px-0 col-12 offset-0 col-lg-8 offset-lg-2">
+                            <input class="form-control w-100 p-4 m-0 d-inline-block mr-5 job-search-input-guest" type="text" name="search" id="searchbox"
+                        placeholder="Search jobs (e.g. Work from Home)">
+                        <a href="#" class="btn btn-primary search-button" name="">Search Jobs </a>
+                        </div>
+                       </div>
+                   </form>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <div class="container mt-5">
+        {{-- Latest Job   --}}
+        <h1 class="mt-5 mb-3 font-weight-bold text-secondary">Latest Job</h1>
+       @if ($jobs->count() > 0)
+            <div class="row">
+                @foreach ($jobs as $index => $job)
+                    @if($index < 8)
+                        <div class="col-lg-3 col-sm-6 khaled">
+                            <div class="card latest-job my-2">
+                                <div class="card-body box-shad">
+                                    <h5 class="card-title"><a href="">{{$job->title}}</a></h5>
+                                    <span class="card-text">{{$job->company->name}}</span>
+                                    <span> - {{$job->city->name}}</span>
+                                    <p class="card-text"><small class="text-muted">{{\Illuminate\Support\Carbon::now()->diffForHumans(\Illuminate\Support\Carbon::parse($job->created_at))}} Ago</small></p>
+                                </div>
+                            </div>
+                        </div>
+                    @endif
+                @endforeach
+            <div class="col-sm-12">
+                <div class="float-right my-3">
+                    <a href="#" >See all new Jobs on Land A Job</a>
+                </div>
+            </div>
```

Comment:
If this is possible and there are no jobs.
don't show this section only, don't tell the user.